agent/balrogagent/test/test_cmd.py

```diff
@@ -0,0 +1,71 @@
+import aiohttp
+import asyncio
+import asynctest
+import json
+
+from ..cmd import run_agent
+
+
+@asynctest.patch("balrogagent.client.request")
+# TODO: why do we have to mock the versions of these imported into cmd isntead
+# of mocking balrogagent.changes.*?
```

Comment:
I think this is OK, because they are imported in `balrogagent.cmd` and you want to patch them (the used ones), not the original ones.